models/Review.js

```diff
@@ -18,16 +18,16 @@ require ('./Freelance');
 const Review = exports.Review = new mongoose.Schema({
 		author 	: { type: String, required: true },
 		text 	: { type: String, default: "No text." },
-		score 	: { type: Integer },
+		score 	: { type: Number },
 		target 	: { type: ObjectID, ref: "Freelance", required: true }
```

Comment:
It's better to add a comma also at the end of the last field so that when adding other lines there are less edits.